### ISSUE TITLE
Not in scope: ‘OP.linebreak’

### ISSUE THREAD

**Issue by xave** [lazamar/haskell-docs-cli#9]
When running `cabal install --global haskell-doc-cli``, I get an error:

```

src/Main.hs:73:42: error:
    Not in scope: ‘OP.linebreak’
    NB: the module ‘Options.Applicative.Help.Pretty’ does not export ‘linebreak’.
   |
73 |   , O.footerDoc $ Just $ moreInfoText <> OP.linebreak
   |                                          ^^^^^^^^^^^^

```

This `linebreak` function is indeed not there.

**Comment by xave** [comment]
I think it should just be: 
```
, O.footerDoc $ Just $ moreInfoText <> OP.line
                                       ^^^^^^^
```

**Comment by lazamar** [comment]
Fixed by  #10
